server/src/main/java/io/druid/segment/realtime/appenderator/Appenderators.java

```diff
@@ -124,8 +124,8 @@ public boolean isAnnounced(DataSegment segment)
         null,
         indexIO,
         indexMerger,
-        null,
-        null
+        MapCache.create(500000),
```

Comment:
Could you please also test this cache configuration missing, e. g. add something to test that would fail before this change? (Also see my comment to `DefaultOfflineAppenderatorFactoryTest`)